guacml/pipeline.py

```diff
@@ -0,0 +1,33 @@
+from guacml.dataset import Dataset
+from guacml.preprocessing.column_analyzer import ColumnAnalyzer
+
+
+class Pipeline:
+
+    def __init__(self, guac, model_name):
+        mr = guac.model_results[model_name]
```

Comment:
If im not mistaken `model_results == None` if `guac.run` wasn't called yet. WDYT about raising a nice error to the user to tell him about this dependency? Another option could be to call `guac.run` in `Pipeline.__init__` if id wasn't run already before and only notify the user about the fact that we did.